### DISCUSSION TITLE
OSP-suite compatible with R4.1?

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#922] [Questions & Problems] [answered]
Dear community, 
I am trying to use the OSPsuite using R4.1.0, following the installation guidelines and thus first installing R6, then rCLr (v.0.9.1) and then finally the ospsuite itself (v.10.0.25), however I get the error that "package "ospsuite"  was installed before R 4.0.0: please re-install it". I tried using an older R version (R4.0.0) but the same error persists. Should I use an older version of the ospsuite (and thus also rCLr) or reinstall R3.6 again? 
Thank you for your insights!

Kind regards,
Robin

**Comment by PavelBal** [comment, marked as answer]
Hi Robin,

check the most recent version of ospsuite (https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r/build/artifacts). Also make sure that all the dependencies   
-R6,
-stringr,
- readr

Are installed.

**Reply by RobinM92** [reply]
Dear Pavel,

Thank you very much, using 10.0.35 solved the issue, I was able to install the r ospsuite for R 4.1.

All the best,
Robin
